### ISSUE TITLE
NVDA is reading as "blank" for all the objects in word document on tab navigation

### ISSUE THREAD

**Issue by WiproPSAccTest** [nvaccess/nvda#9398]
### Steps to reproduce:
1. Open a multipage word document with multiple objects(PIcture/Shape/SmartArt/Chart/Video/icon etc)
2. Select the first object using Ctrl+Alt+5 and use Tab to loop through the different objects
3. Observe That NVDA is just announcing blank for all the objects.

### Actual behavior:
NVDA is reading as 'Blank' for all the floating objects. Object name. type, alt text, wrap type info is unknown to the user.

### Expected behavior:
NVDA should announce all the info(Object name. type, alt text, wrap type, Selected/Non selected) when we navigate to the objects. 

#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
NO

**Comment by Adriani90** [comment]
Screenshots are not readable for blind people actively contributing here.Can you please describe what you did?Von meinem iPhone gesendetAm 08.11.2024 um 12:25 schrieb msftedad ***@***.***>:﻿
Hi @Adriani90, AS verified issue is not repro with NVDA version: 2024.4rc1 (2024.4.0.34248). For reference attached the screenshot.
image.png (view on web)
image.png (view on web)

**Comment by seanbudd** [comment]
Closing as fixed

**Comment by msftedad** [comment]
> Screenshots are not readable for blind people actively contributing here.Can you please describe what you did?Von meinem iPhone gesendetAm 08.11.2024 um 12:25 schrieb msftedad ***@***.***>:﻿ Hi @Adriani90, AS verified issue is not repro with NVDA version: 2024.4rc1 (2024.4.0.34248). For reference attached the screenshot. image.png (view on web) image.png (view on web) —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

@Adriani90 - Sorry for missing this, we will surely take care of it going forward. Though issue closed now, still we have updated the attachment with video.
